app/sidekiq/hca/health_facilities_import_job.rb

```diff
@@ -66,5 +54,32 @@ def get_facilities_from_lighthouse
 
       all_facilities.index_by { |facility| facility[:id] }
     end
+
+    # Ensure the std_states table is populated. The table is populated in a
+    # daily job, but this ensures the table is populated in any environment
+    # that the table may be empty in (review instances, new environment, locally)
+    def ensure_std_states_populated
+      return if StdState.exists?
+
+      ::IncomeLimits::StdStateImport.new.perform
+      raise 'StdStates missing – triggered import and retrying job'
+    end
+
+    # Filter on only health facilities and return hash with
+    # name, station_number (also known as facility_id),
+    # and postal_name (also known as state_code ex OH,MI)
+    def facilities_with_postal_names(facilities_from_lighthouse)
+      StdInstitutionFacility
+        .includes(:street_state)
+        .where(station_number: facilities_from_lighthouse.keys)
+        .pluck(:station_number, 'std_states.postal_name')
+        .map do |station_number, postal_name|
+        {
+          name: facilities_from_lighthouse[station_number][:name],
+          station_number:,
+          postal_name:
+        }
```

Comment:
Your Ruby clean clean.
![a-baby-is-sitting-at-a-table-with-a-bowl-of-food-and-making-a-funny-face](https://github.com/user-attachments/assets/7322f255-9b4b-4563-afc7-4684e7274bfe)